src/extension/ScriptRunner.ts

```diff
@@ -144,11 +152,21 @@ export class ScriptRunner {
    * @returns job id (string)
    */
   private getJobId(link: string): string {
-    const pattern = 'https://(prover|vaas-stg).certora.com/output/'
-    const regExp = new RegExp(pattern)
+    const regExp = this.getUrlReg()
     return link.split('?anonymousKey')[0].replace(regExp, '').split('/')[1]
   }
 
+  /**
+   * return the cvl version number
+   * @param str
+   */
+  private whichVersion(str: string): number {
+    if (str.includes('certora-cli-')) return 2
```

Comment:
please define CVL_VERSION_1 (=1), CVL_VERSION_2(=2) as global consts, then it can be used thorough all files (instead of 1/2)